examples/shadowmap/main.js

```diff
@@ -1137,8 +1146,7 @@
             var shadowSettings;
             if ( this._config[ 'atlas' ] ) {
 
-                shadowSettings = this._shadowSettings[ num ];
-                shadowMap = this._shadowMapAtlas.addLight( light, shadowSettings );
+                shadowMap = this._shadowMapAtlas.addLight( light );
```

Comment:
Not sure it's intended in this function 'addShadowedLight' 2 calls to this._shadowMapAtlas.addLight( light ); are executed if atlas is true.
one at line 1149 and another 1235